free-3.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I imagine this being helpful for novice potery writers who may need help writing poems in certain formats, such as sonnets and limericks, and want various types of help in doing so. Now, the experience may be difficult for writers who have a hard time with meter or with rhyming. I imagine this DSL to suggest words to users to help them with rhyme scheme, point out overused words, and look up pronounciations of words to make sure meter and rhythm is correct.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because poems, and especially structured poems, are a very particular and strictly defined subset of natural language, that can be checked as long as the rules are properly described. I think it addresses the need by acting as a domain-specific spellchecker, to which inputs (which type of poem, the poem itself) can be input, and suggestions can be output.
```

Comment:
Interesting stuff! I feel like this project would also benefit greatly from a simple GUI.